src/main/java/com/google/api/resourcenames/UntypedResourceName.java

```diff
@@ -78,6 +82,22 @@ public ResourceNameType getType() {
     return UntypedResourceNameType.instance();
   }
 
+  @Override
+  /* Returns a map with an empty String as the sole key, which maps to the raw value of this ResourceName. */
+  public Map<String, String> getFieldValuesMap() {
```

Comment:
Put the annotations after the doc comment.

Reply:
yep